RecoTracker/TkDetLayers/src/TkDetUtil.cc

```diff
@@ -40,7 +49,7 @@ namespace tkDetUtil {
       auto y1 = hori ? yc - maxDistance.y() :  xc - maxDistance.x();
       auto x1 = hori ? xc + maxDistance.x() : -yc + maxDistance.y();
 
-      dphi = std::acos( (x0*x1+y0*y1)/std::sqrt((x0*x0+y0*y0)*(x1*x1+y1*y1)) );
+      dphi = clamped_acos((x0*x1+y0*y1)/std::sqrt((x0*x0+y0*y0)*(x1*x1+y1*y1)));
```

Comment:
this is where offtrack is filled
http://cmslxr.fnal.gov/lxr/source/DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc?v=CMSSW_8_0_0_pre3#1007
unless cluster fails 
http://cmslxr.fnal.gov/lxr/source/DQM/SiStripMonitorTrack/src/SiStripMonitorTrack.cc?v=CMSSW_8_0_0_pre3#1055

do not see any correlation with the changes in this PR